crates/agent/src/assistant_panel.rs

```diff
@@ -48,6 +51,13 @@ use crate::{
     OpenActiveThreadAsMarkdown, OpenAgentDiff, OpenHistory, ThreadEvent, ToggleContextPicker,
 };
 
+const ASSISTANT_PANEL_KEY: &str = "assistant_panel";
```

Comment:
We should call this "agent_panel":

```suggestion
const AGENT_PANEL_KEY: &str = "agent_panel";
```

We haven't updated some of our existing naming yet to prevent churn, but for new code we should use "Agent Panel" everywhere.

Reply:
Fixed this